src/engine/Server.cpp

```diff
@@ -170,9 +170,11 @@ Awaitable<json> Server::composeResponseQleverJson(
     j["status"] = "OK";
     j["warnings"] = qet.collectWarnings();
     j["selected"] =
-        query.hasSelectClause()
-            ? query.selectClause()._selectedVariables
-            : std::vector<std::string>{"?subject", "?predicate", "?object"};
+        query.hasSelectClause() ?
```

Comment:
This is wrong,
even with the * you have to put all the variables
in the ["selected"] field, and they have to be in the correct order
(as by the column indices).